### ISSUE TITLE
doesnt have fail limit

### ISSUE THREAD

**Issue by amirmahdi801** [N74NK/BTC#2]
please add a fail limit like dogecrot but with 10 fails maximum

**Comment by ESudhir** [comment]
Yes, there is no fail limit. If a captcha appears, the script keeps failing continuously. Add a fail limit of 10 and press "skip" button if it fails 10 times
![Screenshot_2019-06-26-15-40-06-854](https://user-images.githubusercontent.com/47020178/60171815-0b2b9f00-9829-11e9-8447-9bc55fdd3e67.jpeg)

**Comment by lomadapraveenreddy** [comment]
is this script safe.
